pennylane/beta/tapes/qubit_param_shift.py

```diff
@@ -89,15 +89,15 @@ def _update_circuit_info(self):
         # check if the quantum tape contains any variance measurements
         self.var_mask = [m.return_type is qml.operation.Variance for m in self.measurements]
 
+        # Make a copy of the original measurements; we will be mutating them
+        # during the parameter shift method.
+        self._original_measurements = self._measurements.copy()

```

Comment:
The second-order CV parameter-shift rule needs a copy of the original measurements even if there are no variances.